burglebros.js

```diff
@@ -598,13 +592,15 @@ function (dojo, declare) {
             dojo.destroy('generic_token_' + id);
         },
 
-        createCardToken: function(id, type, count) {
+        createCardToken: function(id, type, count, extra_classes='', draggable=false) {
```

Comment:
I don't think you can use es6 default arguments. It would fail when minified.

Reply:
actually, i think it works with my other games on BGA ;)
but i'll remove the default arguments and pass them in the function calls